interface/modules/custom_modules/oe-module-payroll/public/provider_rate/index.php

```diff
@@ -0,0 +1,118 @@
+<?php
```

Comment:
does this script need an access control?

Reply:
I put the access control in the menu. The menu determines who sees which link. do you think there should be a secondary access control?

Reply:
Recommend matching the acl that controls the menu link  to an acl check on the script itself (this would stop somebody who should not have access to the script by simply logging to openemr and going directly to the script in the browser; and also easy to send get/post parameters via Burp suite)